common/static/coffee/spec/discussion/view/new_post_view_spec.coffee

```diff
@@ -10,21 +10,29 @@ describe "NewPostView", ->
         )
         @discussion = new Discussion([], {pages: 1})
 
-    checkVisibility = (view, expectedVisible, expectedDisabled) =>
-      view.render()
-      expect(view.$(".js-group-select").is(":visible")).toEqual(expectedVisible)
-      disabled = view.$(".js-group-select").prop("disabled")
-      if expectedVisible and ! expectedDisabled
+    checkVisibility = (view, expectedVisible, expectedDisabled, noRender) =>
+      if !noRender
```

Comment:
Rather than a `noRender` argument, where you have to check "if not noRender", it would be clearer to have a `render` argument, and check "if render". Avoiding double-negatives makes it easier to reason about code.